ecs/accounts/templates/home.html

```diff
@@ -1,11 +1,15 @@
 {% extends 'base.html' %}
 
 {% block title %}
+    {% load static from staticfiles %}
```

Comment:
a nie powinienem dać  {% load static from staticfiles %} na wszelki wypadek?
http://whitenoise.evans.io/en/stable/django.html#make-sure-staticfiles-is-configured-correctly